api/CSSMediaRule.json

```diff
@@ -0,0 +1,54 @@
+{
+  "api": {
+    "CSSCounterStyleRule": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSMediaRule",
+        "support": {
+          "webview_android": {
+            "version_added": "45"
+          },
+          "chrome": {
+            "version_added": "45"
+          },
+          "chrome_android": {
+            "version_added": "45"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": true,
+            "notes": "Setting the value of conditionText on Gecko is only possible from Firefox 20 onwards."
+          },
+          "firefox_android": {
+            "version_added": "17",
+            "notes": "Setting the value of conditionText on Gecko is only possible from Firefox 20 onwards."
```

Comment:
As before, I'd redo this note as `Before Firefox 20, <code>conditionText</code> could not be set.`